hr_skill/models/hr_skill.py

```diff
@@ -22,6 +22,7 @@ class Skill(models.Model):
                                          'skill_id',
                                          string='Employees')
     description = fields.Char(string="Description")
+    color = fields.Integer('Color Index', default=10)
```

Comment:
Why is color an integer?

Reply:
That's how Odoo sets it on for tags and display the color selector.

Reply:
@tarteo There are similar code pieces all over Odoo codebase

Reply:
Didn't know that's how it works for tags. Thanks! :+1: